app/translations/sw/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,616 @@
+# German translations for PROJECT.
+# Copyright (C) 2014 ORGANIZATION
+# This file is distributed under the same license as the PROJECT project.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2014.
+#
+msgid ""
+msgstr ""
```

Comment:
This section may need to be updated.

Reply:
Thanks. Updated language name and year. Left PROJECT, ORGANIZATION, FIRST AUTHOR <EMAIL@ADDRESS> as is.